src/app/models/main.rs

```diff
@@ -99,6 +109,12 @@ impl AlbumDescription {
         let duration: u32 = self.songs.iter().map(|song| song.duration).sum();
         format_duration(duration.into())
     }
+    pub fn year(&self) -> Option<u32> {
+        self.release_date
+            .as_ref()
+            .and_then(|date| date.split('-').next())
+            .and_then(|y| u32::from_str(y).ok())
+    }
```

Comment:
So it looks better now

Reply:
oh it was not so much about human friendliness, it's just that calls to unwrap() can panic!, and it's easy enough to avoid them here so...

but yeah that's better!!